test/Sentry.Unity.Tests/DebouncerTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Sentry.Unity.Tests
+{
+    /// <summary>
+    /// Testing real in realtime.
+    /// </summary>
+    public sealed class DebouncerTests
+    {
+        private readonly TimeSpan DefaultOffset = TimeSpan.FromSeconds(1);
```

Comment:
Ideally we mock any 'time based' test as they tend to:

1. Slow down builds
2. Be flaky

If there's no way around it, can we rely on timing such as:

```suggestion
        private readonly TimeSpan DefaultOffset = TimeSpan.FromMilliseconds(1);
```

Reply:
Made 100ms